definitions/PaperDollItemFrame.dbd

```diff
@@ -24,3 +24,8 @@ $id$ID<32>
 ItemButtonName
 SlotIcon
 SlotNumber<32>
+
+BUILD 0.5.3.3368
+ItemButtonName
```

Comment:
This not having an ID would fix that 6.0.1 had this as padding, i.e. id-but-ignored. Again, can we verify this in file?

Reply:
Looks fine
![image](https://user-images.githubusercontent.com/50212/37566537-cc8c140a-2aba-11e8-883e-aa94b29407be.png)